github/repos_comments.go

```diff
@@ -109,6 +116,9 @@ func (s *RepositoriesService) GetComment(owner, repo string, id int) (*Repositor
 		return nil, nil, err
 	}
 
+	// TODO: remove custom Accept header when this API fully launches.
+	req.Header.Set("Accept", mediaTypeReactionsPreview)
+
 	c := new(RepositoryComment)
 	resp, err := s.client.Do(req, c)
 	if err != nil {
```

Comment:
Do you know if `UpdateComment` needs to include the custom Accept header too, or not?